neqo-http3/src/connection_client.rs

```diff
@@ -1952,6 +1969,100 @@ mod tests {
         client.close(now(), 0, "");
     }
 
+    #[test]
+    fn multiple_goways() {
+        let (mut client, mut server) = connect();
+        let request_stream_id_1 = make_request(&mut client, false);
+        assert_eq!(request_stream_id_1, 0);
+        let request_stream_id_2 = make_request(&mut client, false);
+        assert_eq!(request_stream_id_2, 4);
+        let request_stream_id_3 = make_request(&mut client, false);
+        assert_eq!(request_stream_id_3, 8);
+
+        let out = client.process(None, now());
+        server.conn.process(out.dgram(), now());
+
+        // First send Goaway with an higher number
+        let _ = server
+            .conn
+            .stream_send(server.control_stream_id.unwrap(), &[0x7, 0x1, 0x8]);
+
+        let out = server.conn.process(None, now());
+        client.process(out.dgram(), now());
+
+        // Check that there is one reeset for stream_id 8
```

Comment:
```suggestion
        // Check that there is one reset for stream_id 8
```

Reply:
My keyboard starts to have problems with e as well. It started to write double e. (and it already have the same problem with 't') this is not good.